src/sentry/stacktraces/processing.py

```diff
@@ -273,16 +273,32 @@ def _has_system_frames(frames):
     return bool(system_frames) and len(frames) != system_frames
 
 
-def _normalize_in_app(stacktrace: Sequence[dict[str, str]]) -> None:
+def _normalize_in_app(stacktrace: Sequence[dict[str, str]]) -> str:
     """
-    Ensures consistent values of in_app across a stacktrace.
+    Ensures consistent values of in_app across a stacktrace. Returns a classification of the
+    stacktrace as either "in-app-only", "system-only", or "mixed".
```

Comment:
Please add a note that it will be used for tracking purposes. It's hard to know the purpose of the code without a mention of the why (well, one could end up going to the PR to discover it).

Reply:
Done here: https://github.com/getsentry/sentry/pull/54453/files#diff-a5fb5bd743c57f79634528b0f94eef6cbe79295ab61d1f5051f310b8b5323442R278-R279

https://github.com/getsentry/sentry/blob/99f7937bea5d6d7851f55c968804fe8406d60ab2/src/sentry/stacktraces/processing.py#L278-L279